specs/latest/2.0/index.html

```diff
@@ -1073,15 +1167,17 @@ <h2 class="no-toc">Table of contents</h2>
         See <a href="#COPYING_BUFFERS">Copying Buffers</a> for restrictions imposed by the WebGL 2 API.
       </dd>
       <dt>
-        <p class="idl-code">void getBufferSubData(GLenum target, GLintptr offset, ArrayBuffer? returnedData)
+        <p class="idl-code">
+          void getBufferSubData(GLenum target, GLintptr offset,
+                                ByteBuffer returnedData,
+                                optional unsigned long byteOffset = 0)
```

Comment:
Also, for this one perhaps the naming convention can be "returnedOffset" and "returnedSize".